src/sql/base/browser/ui/editableDropdown/dropdown.ts

```diff
@@ -298,7 +298,7 @@ export class Dropdown extends Disposable {
 		this.$treeContainer.style('outline', `1px solid ${style.contextBorder || this._options.contextBorder}`);
 	}
 
-	private _inputValidator(value: string): IMessage {
+	private _inputValidator(value: string): IMessage | null {
```

Comment:
should we use undefined here?  I thought we wanted to use prefer undefined to null through out the codebase.

Reply:
null is being adopted from a vscode constraint (they define it as null).